src/website/shared/models/nix_evaluation.py

```diff
@@ -341,36 +341,46 @@ def channel_structure(derivations: list[NixDerivation]) -> dict:
         # FIXME This is wrong. Replace with something like builtins.parseDrvName
         version = derivation.name.split("-")[-1]
         if attribute not in packages:
-            packages[attribute] = dict()
+            packages[attribute] = {
+                "versions": {},
```

Comment:
Nit, maybe it should be `nixpkgs-releases` instead of versions, for clarity?

Reply:
When I hear nixpkgs release I think of 24.04 or unstable, but this is naming the data structure that contains package versions, grouped by nixpkgs releases.

Reply:
Ah then I misread, sorry.